sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -229,7 +229,9 @@ export class ServiceBusClient {
    *  - ApplicationTokenCredentials
    *  - UserTokenCredentials
    *  - DeviceTokenCredentials
-   *  - MSITokenCredentials.
+   *  - MSITokenCredentials 
+   * Token audience to use when creating the credentials is https://servicebus.azure.net/
+   * For MSI based credentials, this must be set on `resource` parameter to the `credentials` object
```

Comment:
Resource can be set when creating the credentials itself right?

> Token audience (or resource in case of MSI based credentials) to use when creating the credentials is https://servicebus.azure.net/